### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by icewind1991 — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]

File: apps/user_ldap/templates/part.wizard-loginfilter.php [hunk 1]
```diff
@@ -9,7 +9,7 @@
 			</label>
 
 			<input type="checkbox" id="ldap_loginfilter_username"
-				   title="<?php p($l->t('Allows login against the LDAP / AD username, which is either uid or samaccountname and will be detected.'));?>"
+				   title="<?php p($l->t('Allows login against the LDAP / AD username, which is either UID or samaccountname and will be detected.'));?>"
```
Comment (blizzz) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: lowercase 'uid' is the technically correct term, I'd prefer to leave it, although it does not make a difference in the end. Rather ``samaccountname`` could be changed to ``sAMAccountName``.

File: apps/user_ldap/templates/settings.php [hunk 2]
```diff
@@ -84,7 +84,7 @@
...
 				<p><label for="ldap_backup_port"><?php p($l->t('Backup (Replica) Port'));?></label><input type="number" id="ldap_backup_port" name="ldap_backup_port" data-default="<?php p($_['ldap_backup_port_default']); ?>"  /></p>
-				<p><label for="ldap_override_main_server"><?php p($l->t('Disable Main Server'));?></label><input type="checkbox" id="ldap_override_main_server" name="ldap_override_main_server" value="1" data-default="<?php p($_['ldap_override_main_server_default']); ?>"  title="<?php p($l->t('Only connect to the replica server.'));?>" /></p>
+				<p><label for="ldap_override_main_server"><?php p($l->t('Disable Main Server'));?></label><input type="checkbox" id="ldap_override_main_server" name="ldap_override_main_server" value="1" data-default="<?php p($_['ldap_override_main_server_default']); ?>"  title="<?php p($l->t('Only connect to the Replica server.'));?>" /></p>
```
Comment (blizzz) [inline apps/user_ldap/templates/settings.php]: no, keep it lower case , "replica" is not a product/name.

**Review by blizzz — CHANGES_REQUESTED** [review]

File: apps/user_ldap/templates/part.wizard-loginfilter.php (hunk 1 above)
Comment (nickvergessen) [inline apps/user_ldap/templates/part.wizard-loginfilter.php]: done

File: apps/user_ldap/templates/settings.php (hunk 2 above)
Comment (nickvergessen) [inline apps/user_ldap/templates/settings.php]: done

**Review by blizzz — APPROVED** [review]

**Comment by blizzz** [comment]
Some test adjustments left

File: apps/files_external/lib/Controller/UserGlobalStoragesController.php [hunk 3]
```diff
@@ -162,15 +162,15 @@ public function update(
 			} else {
 				return new DataResponse(
 					[
-						'message' => (string)$this->l10n->t('Storage with id "%i" is not user editable', array($id))
+						'message' => (string)$this->l10n->t('Storage with ID "%i" is not user editable', array($id))
```
Comment (MorrisJobke) [inline apps/files_external/lib/Controller/UserGlobalStoragesController.php]: Any reason why the `%i` is still here?

**Comment by MorrisJobke** [comment]
> Some test adjustments left

I fixed them. Let's see if those are all :)

**Review by MorrisJobke — APPROVED** [review]
Looks good now 👍

**Pull request merged**